gui.py

```diff
@@ -110,19 +113,62 @@ def wrap_in_section(content, section_classes="", container_classes="", div_class
     section_classes="graph",
 )
 
-data = wrap_in_section(
+
+tool_info = wrap_in_section(
     [
+        html.H3("How the Tool Works", className="title is-4"),
         ddsih.DangerouslySetInnerHTML(
             f"""
-<p>Index based on 1981&ndash;2010 daily normals and standard deviations of 25 stations.  A value of +1 means that the day is warmer than 10&#37; of all Above Normal days.  A value of +8 means that the day is warmer than 80&#37; of all Above Normal days, and so on.  The opposite is true for negative numbers.</p>
-<p>Data provided by the the <a href="http://www.rcc-acis.org">Applied Climate Information System (ACIS)</a>.</p>
-<p>Placeholder, credits & citation.</p>
+<p>The tool compares reliable observations from a network of stations distributed across the state to baseline
+normals collected and averaged over the three-decade period from 1981 to 2010. Data is collected from the National
+Weather Service’s Automated Surface Observing Systems (ASOS). This system includes mean and standard deviations of
+daily normal temperatures, and covers most of the state.</p>
+<p>Map of the ASOS stations used to determine the Statewide Temperature Index.</p>
             """
-        )
+        ),
+        html.Figure(
+            children=[
+                html.Img(
+                    height="480px",
+                    width="600px",
+                    src=path_prefix + "assets/asos_station_map.jpg",
+                )
+            ],
+        ),
+        ddsih.DangerouslySetInnerHTML(
+            f"""
+<p>Utilizing this network allows for the geographic and latitudinal variation inherent to the state of Alaska to be
+taken into account without a large degree of complexity.</p>
+<p>Advantages of a daily temperature index</p>
+<ul>
+    <li>It is not strongly influenced by occasional missing data points</li>
+    <li>It can distinguish moderate anomalies in statewide data. Additionally, a single number is easy to understand and disseminate.</li>
+</ul>
+<p>Other considerations</p>
+<ul>
+    <li>A single Index number can make the data easy to misunderstand, and makes it challenging to quantify extreme temperature variations.</li>
+    <li>Production of the index using the ASOS system also means that the index has the same gaps in its regional coverage as that system. The ASOS system is subject to occasional sensor failures, as well as failures in communication systems. There can be some lag between failure and repair.</li>
+</ul>
+            """
+        ),
+    ],
+    div_classes="content is-size-5",
```

Comment:
Add this to the app css for consistent widths for text blocks (not perfect but it works here):
```
.content {
	max-width: 40em;
}
```